fastlane/lib/fastlane/actions/set_info_plist_value.rb

```diff
@@ -10,7 +10,10 @@ def self.run(params)
         begin
           path = File.expand_path(params[:path])
           plist = Plist.parse_xml(path)
-          if params[:subkey]
+          if params[:map]
+            stringified_hash = Hash[params[:map].collect { |k, v| [k.to_s, v] }]
```

Comment:
`plist` from `Plist.parse_xml` is stringified, and merge cannot merge symbols and strings it duplicates them.
So we have to convert it to keyed strings.